### DISCUSSION TITLE
IP ban permission

### DISCUSSION THREAD

**Discussion by enno123** [NamelessMC/Nameless#2159] [Ideas]
If a user is banned with the spam button in the forum, this ban will not be listed in the penalty list.
Furthermore you can ban yourself with the button.
http://prntscr.com/s239uk

How can I turn off the IP ban for moderators? So that they can only ban normally.

**Comment by samerton** [comment]
Unfortunately there's currently no IP ban permission, I can look into adding one though

**Comment by enno123** [comment]
Is it possible to add a normal ban button and or warning button? Because if you want to ban or warn a player you have to go to the AdminCP first and then search for the user
